gen3-cli/cot-testcase.sh

```diff
@@ -0,0 +1,3 @@
+#! /bin/bash
```

Comment:
although this space after the `!` character could work in some cases I believe it should not be there